lib/view/sfViewCacheManager.class.php

```diff
@@ -442,6 +442,8 @@ public function remove($internalUri, $hostName = '', $vary = '', $contextualPref
         if ($this->cache->has($cacheKey)) {
             return $this->cache->remove($cacheKey);
         }
+
+        return false;
```

Comment:
I would return `true` here, as if the key is not there, it is like is has been removed successfully.

Wdyt?

Reply:
true. i mean you are right.